.gitmodules

```diff
@@ -1,9 +1,9 @@
 [submodule "rocksdb"]
 	path = librocksdb_sys/rocksdb
-	url = https://github.com/tikv/rocksdb.git
-	branch = 6.29.tikv
+	url = https://github.com/tabokie/rocksdb.git
+	branch = 221215-post-write
 
 [submodule "titan"]
 	path = librocksdb_sys/libtitan_sys/titan
-	url = https://github.com/tikv/titan.git
-	branch = master
+	url = https://github.com/tabokie/titan.git
```

Comment:
ticker enum should be updated in rust-rocksdb if titan is updated due to https://github.com/tikv/titan/pull/262